src/Microsoft.IdentityModel.Clients.ActiveDirectory/AuthenticationResult.cs

```diff
@@ -122,9 +122,9 @@ internal AuthenticationResult(string accessTokenType, string accessToken, DateTi
         public string IdToken { get; internal set; }
 
         /// <summary>
-        /// Gets the authority which has issued tokens.
+        /// Gets the authority that has issued the token.
         /// </summary>
-        internal string Authority { get; set; }
```

Comment:
This is an addition to the public API. We need to increase the minor number of the release (3.18.0, not 3.17.4)

Reply:
I increased the minor number to 3.18.0